### PR TITLE
Cordova 4.0 Updates

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: samples/readme.md [hunk 1]
```diff
@@ -2,11 +2,12 @@
 <pre>
 cordova create $appname com.salesforce.$template $appname
 cd $appname
-cordova plugin add org.apache.cordova.contacts                                          (contactexplorer only)
-cordova plugin add org.apache.cordova.statusbar                                         (contactexplorer only)
+cordova plugin add cordova-plugin-contacts                                              (contactexplorer only)
```
Comment (bhariharan) [inline samples/readme.md]: New format of specifying plugins.

File: samples/readme.md [hunk 2]
```diff
@@ -2,11 +2,12 @@
...
+cordova plugin add cordova-plugin-statusbar                                             (contactexplorer only)
 cordova plugin add https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin
+cordova plugin add cordova-plugin-whitelist
```
Comment (bhariharan) [inline samples/readme.md]: New mandatory ```Whitelist``` plugin.

File: test/MockSDKInfo.js [hunk 3]
```diff
@@ -49,8 +49,8 @@ var MockSDKInfo = (function(window) {
                                       ["com.salesforce.oauth", "com.salesforce.sdkinfo", "com.salesforce.sfaccountmanager", "com.salesforce.testrunner", "com.salesforce.smartstore", "com.salesforce.smartsync"], 
                                       "ForcePluginsTest", "1.0",
                                       {
-                                          "remoteAccessConsumerKey": "3MVG92.uWdyphVj4bnolD7yuIpCQsNgddWtqRND3faxrv9uKnbj47H4RkwheHA2lKY4cBusvDVp0M6gdGE8hp",
-                                          "oauthRedirectURI": "sfdc:///axm/detect/oauth/done",
+                                          "remoteAccessConsumerKey": "3MVG9Iu66FKeHhINkB1l7xt7kR8czFcCTUhgoA8Ol2Ltf1eYHOU4SqQRSEitYFDUpqRWcoQ2.dBv_a1Dyu5xa",
```
Comment (bhariharan) [inline test/MockSDKInfo.js]: I'm not sure why these guys were pointing to the old ```Chatter``` values - fixed them.

**Comment by bhariharan** [comment]
I removed the ```whitelist``` plugin from the instructions and added it as a dependency in our plugin directly, so it's always pulled in.

**Comment by wmathurin** [comment]
:+1:

**Pull request merged**